peer.go

```diff
@@ -368,25 +368,30 @@ out:
 		var isChanUpdate bool
 		var targetChan *wire.OutPoint
 
+		peerAddress := &lnwire.NetAddress{
```

Comment:
Rather than passing a _new_ instane of the peer's address (and also encuring the GC overhead of a new object on _each_ message), I think we should instead pass in the active version (`p.addr`).